src/site/xdoc/server/config.xml

```diff
@@ -39,6 +39,9 @@
     some back-ends, to some protocols. Please also note that some configuration files are not required for Guice. In this
     case, it will be specified.</p>
        
+    <p>In every configuration files, you can use the following notation to use an environment variable: <code>${env:MY_VAR}</code>.
+    If <code>MY_VAR</code> exists, it will be replaced by the matching environment variable.</p>
+    
```

Comment:
I think you should add an explanation if  `<code>${env:MY_VAR}</code>` does not exist, so end users don't confuse  whether the result is `null` or `${env:MY_VAR}` plain string

Reply:
good point, done